python/tests/res/enkf/CMakeLists.txt

```diff
@@ -79,9 +79,9 @@ addPythonTest(tests.res.enkf.test_log_config.LogConfigTest)
 addPythonTest(tests.res.enkf.test_run_context.ErtRunContextTest)
 addPythonTest(tests.res.enkf.test_enkf_runpath.EnKFRunpathTest)
 addPythonTest(tests.res.enkf.test_programmatic_res_config.ProgrammaticResConfigTest)
+addPythonTest(tests.res.enkf.test_enkf.EnKFTest)
 
 if (STATOIL_TESTDATA_ROOT)
-  addPythonTest(tests.res.enkf.test_enkf.EnKFTest LABELS StatoilData)
```

Comment:
This test is not using StatoilData, I see no reason why it should be dependent on STATOIL_DATA_ROOT

Reply:
Well - then it should probably be moved out of the `STATOIL_DATA_ROOT` section. We have gradually tried to reduce the Statoil data grip on the tests, so my assumption is that it used statoil data previously.